capstone/src/main/webapp/js/autocomplete.js

```diff
@@ -0,0 +1,140 @@
+const inp = document.getElementById('search');
+const currentFocus = -1;
+/**
+ * @param {Object} autocompleteListElmt
+ * @return {bool}
+ * Adds the 'active' tag to an autocomplete elmt
+ */
+function addActiveTag(autocompleteListElmt) {
+  if (!autocompleteListElmt) {
+    return false;
+  }
+  removeActiveTag(autocompleteListElmt);
+  if (currentFocus >= autocompleteListElmt.length) {
+    currentFocus = 0;
+  } else if (currentFocus < 0) {
+    currentFocus = (autocompleteListElmt.length - 1);
+  }
+  autocompleteListElmt[currentFocus].classList.add('autocomplete-active');
```

Comment:
Seems like `autocomplete-active` is a good candidate for a constant!

Reply:
will do!